pykafka/producer.py

```diff
@@ -434,17 +434,27 @@ def get_delivery_report(self, block=True, timeout=None):
 
     def _produce(self, message):
         """Enqueue a message for the relevant broker
-
+		Incase of  'success = False':
```

Comment:
Wording can be improved: `Attempts to update metadata in response to missing brokers.`